### DISCUSSION TITLE
bcv data set dice is all 0

### DISCUSSION THREAD

**Discussion by nebstudio** [MIC-DKFZ/nnUNet#878] [Q&A] [answered]
When the new version of nnunet runs the bcv data set, the dice is all 0. What is the reason?

**Comment by FabianIsensee** [comment, marked as answer]
have you verified that the data got preprocessed correctly? Please run nnUNet_plot_task_pngs

**Reply by nebstudio** [reply]
Okay, thank you for your answer, I’m sure it’s the problem of dealing with data sets.
